kubernetes/pro/application/api-server-deployment.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: api-server
+  name: api-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: api-server
+  strategy: {}
+  template:
+    metadata:
+      labels:
+        app: api-server
+    spec:
+      containers:
+        - env:
+            - name: IS_PRO
+              value: "true"
+            - name: Kafka__BootstrapServers
+              value: kafka:9092
+            - name: MongoDb__ConnectionString
+              value: mongodb://admin:password@mongodb:27017
+            - name: MongoDb__Database
+              value: featbit
+            - name: OLAP__ServiceHost
+              value: http://da-server
```

Comment:
it should be http://da-server:8200 according to da-server-service.yml

Reply:
fixed with 9e6a819